### DISCUSSION TITLE
Programmatically open popover next to an element [Dynamic target element]

### DISCUSSION THREAD

**Discussion by guyschlider** [Akryum/v-tooltip#708] [Q&A]
Hey,

Is it possible to have a single Popover element that will be programmatically positioned next to a target element?

Could not find an example while reading the documentation nor the issues

**Comment by Zulqarnain-Haider** [comment]
You can use container property of the v-popover like this **container="#targetElement"**.

Example

```
 <v-popover :disabled="!isEnabled"  placement="left"  container="#targetElement" :offset="10">
 <template slot="popover">
 popover
 </template>
 </v-popover>
```

**Comment by lincolnlemos** [comment]
This does not work for me... When I add the props container="#targetElement" the Popper is being appended to the #targetElement, but the position is still being evaluated based on the trigger element.

Looking to the source code, https://github.com/Akryum/v-tooltip/blob/v2.1.3/src/components/Popover.vue#L396
Looks like there are no options to update the reference?

Any ideas @Zulqarnain-Haider  @Akryum?

Here is my not working example
https://jsfiddle.net/bsdj6gLz/
